pkg/parser/parser.y

```diff
@@ -281,8 +282,9 @@ import (
 	trailing          "TRAILING"
 	trigger           "TRIGGER"
 	trueKwd           "TRUE"
-	unique            "UNIQUE"
+	tidbCurrentTSO    "TiDB_CURRENT_TSO"
```

Comment:
Should it be consistend and be all upper case?
```suggestion
	tidbCurrentTSO    "TIDB_CURRENT_TSO"
```

Reply:
Yes, maybe. But that's outside of the scope of my PR as the objective of my PR is to reorder without changing anything else.